hack/verify-govet.sh

```diff
@@ -21,8 +21,7 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false \
-              $test_dir
+  go vet -shadow=false $test_dir
```

Comment:
It doesn't seem that `go vet` can use its flags - you still need to use `go tool vet`. Also -shadow is by default false.